.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+# Pull Request \<Title>
+`<Description here>`
+## Things to do during Peer Review
+Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
+- [ ] The code as been manually inspected
```

Comment:
Typo
Is: The code as been manually inspected
Should: The code has been manually inspected

Reply:
fixed